include/rocksdb/options.h

```diff
@@ -381,9 +381,11 @@ struct ColumnFamilyOptions {
 
   // Compression algorithm that will be used for the bottommost level that
   // contain files. If level-compaction is used, this option will only affect
-  // levels after base level.
+  // levels after base level. This setting overides the compression settings
+  // configured via Options.compression but only for the bottommost level.
   //
-  // Default: kDisableCompressionOption (Disabled)
+  // Default: kDisableCompressionOption (Disabled). The means that the setting
```

Comment:
nit:
`The means` => `This means`
minor suggestion:
`the setting` => `the compression`